docs/unity/features/fps-capture.md

```diff
@@ -0,0 +1,49 @@
+---
+title: FPS Capture
+description: FPS Capture for the Embrace Unity SDK
+sidebar_position: 19
+---
+
+# Overview
+
+In mobile games and apps it is crucial to know when/if your app is having FPS issues. Typically you want a target minimum frame rate for your app and our automatic FPS capture tool will log periodically how your app is doing and if you went below a target threshold.
+
+## Getting Started
+
+Start by going to **Tools > Embrace > Settings**
+
+Under the **Settings** Tab enable:
+**Auto Instrumentation FPS Capture**
+
+Please note this is an opt-in feature.
```

Comment:
Can you add as a markdown note item, with a little more context? let me know if the below works

```suggestion
::: Note 
FPS capture is an opt-in feature, meaning it will only work if you enable it using the steps above
:::
```